DSharpPlus/Entities/DiscordEmoji.cs

```diff
@@ -189,7 +194,7 @@ public static DiscordEmoji FromName(DiscordClient client, string name)
                 throw new ArgumentNullException(nameof(name), "Name cannot be empty or null.");
 
             if (UnicodeEmojis.ContainsKey(name))
-                return new DiscordEmoji { Discord = client, Name = UnicodeEmojis[name] };
+                return new DiscordEmoji { Discord = client, Name = UnicodeEmojis[name], RealName = UnicodeEmojis.Keys.FirstOrDefault(k => k == name) };
```

Comment:
Nitpick: use a map lookup here. it faster

Reply:
actually, can't you just use `RealName = name`? what you're doing seems very redundant

Reply:
I just wanted to make sure the key exists to prevent potencial(?) issues, but doing `RealName = name` should do the trick i suppose

Reply:
because it already looks up the map right beforehand there really arent any issues to prevent, i suggest you just do it

Reply:
Done, changed the necessary stuff, and youre right, i was thinking the same way before but i still did the map check for whatever reason 🤦‍♂️